### PR TITLE
Require log in to create haiku

### PR DESCRIPTION
**Pull request by thomas-jung**

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -7,4 +7,8 @@ class ApplicationController < ActionController::Base
   def current_user
     @current_user ||= User.find(session[:user_id]) if session[:user_id]
   end
+
+  def logged_in?
```
Comment (thomas-jung) [inline app/controllers/application_controller.rb]: I did not put this on line 3 helper_method. It seems like this is deprecated. Please let me know if I am mistaken.

**Comment by charliemcelfresh** [comment]
:hamburger:

**Comment by kangkyu** [comment]
haikus spec was also testing not to create a haiku if line content empty, so when logged in, both of the examples can be tested.

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (kangkyu) [inline app/controllers/application_controller.rb]:
```rb
helper_method :current_user, :logged_in?
```
Do you mean this one does not work?

**Comment by thomas-jung** [comment]
I see what you mean. I made the changes and also included logged_in? method in the helper_method. I understand what that is for now and we do need it. Tests are passing. I will merge :smile:
